src/App.tsx

```diff
@@ -1,45 +1,46 @@
-import UserList from './DshboardModule/Components/Users/UserList';
-import ADS from './DshboardModule/Components/ADS/ADS';
-import Booking from './DshboardModule/Components/Booking/Booking';
-import ForgotPassword from './AuthModule/Components/ForgotPassword/ForgotPassword';
-import Register from './AuthModule/Components/Register/Register';
-import ResetPassword from './AuthModule/Components/ResetPassword/ResetPassword';
-import UserLayout from './userLayout/UserLayout';
+import FacilitiesList from "./DshboardModule/Components/FacilitiesList/FacilitiesList";
+import UserList from "./DshboardModule/Components/Users/UserList";
+import ADS from "./DshboardModule/Components/ADS/ADS";
+import Booking from "./DshboardModule/Components/Booking/Booking";
+import ForgotPassword from "./AuthModule/Components/ForgotPassword/ForgotPassword";
+import Register from "./AuthModule/Components/Register/Register";
+import ResetPassword from "./AuthModule/Components/ResetPassword/ResetPassword";
+import UserLayout from "./userLayout/UserLayout";
 import Landing from "./userLayout/Landing";
 import AuthLayout from "./ShareModule/Components/AuthLayout/AuthLayout";
 import NotFound from "./ShareModule/Components/NotFound/NotFound";
 import Layout from "./ShareModule/Components/Layout/Layout";
 import Login from "./AuthModule/Components/Login/Login";
 import MasterLayout from "./ShareModule/Components/MasterLayout/MasterLayout";
 import Favorites from "./userLayout/Favorites/Favorites";
-
 import ExplorePage from "./userLayout/ExplorePage/ExplorePage";
-
-
-import Home from './Home/Home';
-import { DashBoardRoom } from './DashBoardRoomContext/DashBoardRoom';
+import Home from "./Home/Home";
+import { DashBoardRoom } from "./DashBoardRoomContext/DashBoardRoom";
 import RoomDetails from "./userLayout/RoomDetails/RoomDetails";
-
-
-
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+import Checkout from "./userLayout/payment/Checkout";
+const stripePromise = loadStripe(
+  "pk_test_51OTjURBQWp069pqTmqhKZHNNd3kMf9TTynJtLJQIJDOSYcGM7xz3DabzCzE7bTxvuYMY0IX96OHBjsysHEKIrwCK006Mu7mKw8"
+);
 function App() {
   const routes = createBrowserRouter([
-
     {
-      path: '/',
-      element: (<UserLayout />),
+      path: "/",
+      element: <UserLayout />,
       errorElement: <NotFound />,
       children: [
         { index: true, element: <Landing /> },
-        { path: 'landing', element: <Landing /> },
-        { path: 'explor/:state', element: <Landing /> },
-        { path: 'explore', element: <ExplorePage/> },
-        { path: 'Favorites', element: <Favorites/> },
-        { path: 'RoomDetails/:id', element: <RoomDetails/> },
+        { path: "landing", element: <Landing /> },
+        { path: "explor/:state", element: <Landing /> },
+        { path: "explore", element: <ExplorePage /> },
+        { path: "Favorites", element: <Favorites /> },
+        { path: "RoomDetails/:id", element: <RoomDetails /> },
```

Comment:
it is not good to mix between upper and lower case, we have pages that start with lower case and others with upper case
Also we can make `RoomDetails` to `room-details` as I mentioned before